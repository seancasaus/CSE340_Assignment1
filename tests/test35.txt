iqrKz d , PTIkM ((((c)|(4))|(8))|((6)|((a)|(H)))).(((((D).(T))|(Y)).(((8).(3)).(8)))|((((F)|(0)).(m))|(L))) , vZ ((((n)|(k))|(1))|((M)|((C)|(A))))|((((i).(s)).(((6)|(m))|((B).(r))))|((M).(k))) , IggZn E , njqhr ((6).(((J)|(y))|(((i)|(2))|(c)))).((v)|(n)) , Qx (M)|((i).((H).(l))) , ESfY7 u , mgKXs u , WkGXI (B)|((5)|(((8).(n)).(Q))) , t1Zgo ((N).((U).(((E)|(k)).(n)))).((((v).(c)).((C).(W))).(R)) #
" uuuNUEnvcCWRE80muaY838ENUknvcCWRNUEnvcCWR1HY838HDT838NUEnvcCWRiHl6Jn8nQ6JvE80mNUEnvcCWRiHliHlcnUWvNCRk62vNUEnvcCWR1u8Y838aY838iHl5NUknvcCWRuaDT838dBuuBdMuMEiHlEuNUEnvcCWR62vENUEnvcCWRuiHl6LuddiHlcDT8386yvQdNUknvcCWR516JvAiHlu8nQisBr6JvuEM6cvuudEu6yn " 