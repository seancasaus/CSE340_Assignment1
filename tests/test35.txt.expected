ESfY7 , "u"
ESfY7 , "u"
ESfY7 , "u"
t1Zgo , "NUEnvcCWR"
IggZn , "E"
PTIkM , "80m"
ESfY7 , "u"
PTIkM , "aY838"
IggZn , "E"
t1Zgo , "NUknvcCWR"
t1Zgo , "NUEnvcCWR"
vZ , "1"
PTIkM , "HY838"
PTIkM , "HDT838"
t1Zgo , "NUEnvcCWR"
Qx , "iHl"
njqhr , "6Jn"
WkGXI , "8nQ"
njqhr , "6Jv"
IggZn , "E"
PTIkM , "80m"
t1Zgo , "NUEnvcCWR"
Qx , "iHl"
Qx , "iHl"
ERROR
